### PR TITLE
aws-c-http: add version 0.7.11

### PR DESCRIPTION
**Pull request by holger-detering**
Specify library name and version:  **aws-c-http/0.7.11**
---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by gjasny** [comment]
Hello Holger,
could you please touch the PR branch to trigger a new CI run?
Thanks,
Gregor

**Comment by holger-detering** [comment]
PR #19763 should fix the build error on macos / apple-clang with shared libs.

**Comment by gjasny** [comment]
@RubenRBS Do you see what failed here?

**Comment by uilianries** [comment]
@gjasny Sorry the delay. There was an internal build error but I just restarted it now. It should start in few minutes.

**Comment by uilianries** [comment]
I just triggered an internal CI build to generate the missing package and another to rebuild this entire PR. Sorry about this incident.

File: recipes/aws-c-http/all/conanfile.py [hunk 1]
```diff
@@ -40,12 +40,18 @@ def layout(self):
...
     def requirements(self):
-        self.requires("aws-c-common/0.8.2", transitive_headers=True, transitive_libs=True)
-        self.requires("aws-c-compression/0.2.15")
-        if Version(self.version) < "0.6.22":
-            self.requires("aws-c-io/0.10.20", transitive_headers=True)
+        if Version(self.version) <= "0.6.22":
+            self.requires("aws-c-common/0.8.2", transitive_headers=True, transitive_libs=True)
+            self.requires("aws-c-compression/0.2.15")
         else:
+            self.requires("aws-c-common/0.9.0", transitive_headers=True, transitive_libs=True)
```
Comment (uilianries) [inline recipes/aws-c-http/all/conanfile.py:47]:
```suggestion
            self.requires("aws-c-common/0.9.6", transitive_headers=True, transitive_libs=True)
```

**Pull request closed without merging**